integrations/kube-agent-updater/cmd/teleport-kube-agent-updater/main.go

```diff
@@ -106,7 +112,16 @@ func main() {
 		ctrl.Log.Error(trace.BadParameter("--agent-namespace empty"), "agent-namespace must be provided")
 		os.Exit(1)
 	}
+	if versionServer == "" && proxyAddress == "" {
```

Comment:
should we also validate that both aren't empty?

Reply:
Having both set is a valid configuration (and the default one for most clusters). We will try first via the proxy addr, then fallback to the version server if the proxy responds but does not support the new agent AUs.